content/en/universal_service_monitoring/_index.md

```diff
@@ -56,6 +56,7 @@ If you have feedback about what platforms and protocols you'd like to see suppor
 
 - If on Linux:
     - Your service is running in a container.
+    - For non-containerized services see instructions [here][7]
```

Comment:
This is the format we use for anchor links (though your way also works in my local build, not sure how it is translated in the build)
```suggestion
    - For non-containerized services see the [instructions here](#support-for-non-containerized-services).
```